### ISSUE TITLE
Vizy in Super table in Matrix, cannot add fields to Layout

### ISSUE THREAD

**Issue by bliswebagency** [verbb/vizy#46]
**Description**

We have a Super Table field within a Matrix field, as part of a content builder configuration.

When adding a Vizy field to that Super Table field, we can create a group and a block in the Block Configuration panel. However after adding fields to that block's layout, and pressing save, the fields are not saved within that layout.

When using Vizy as standalone field in the same craft install the fields save with the layout. But when we have this Matrix -> Super Table -> Vizy set up then the fields do not save.

**Steps to reproduce**
1. Create Matrix Field
2. Add Super Table field as child of matrix field
3. Add Vizy field as child of super table field
4. Add Group, Block, Layout and add fields to Vizy field
5. Save

**Additional info**
- Plugin version: 1.0.2
- Craft version: 3.6.16
- Multi-site: No

**Additional context**
PHP Version: 7.4.18

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)

**Comment by bliswebagency** [comment]
Awesome. Thanks!
